### PR TITLE
Fixed #24926 -- Removed outdated information from deployment

### PR DESCRIPTION
**Pull request by yamila-moreno**
Fixing this bug: https://code.djangoproject.com/ticket/24926

### CODE REVIEW

File: docs/howto/deployment/index.txt [hunk 1]
```diff
@@ -3,8 +3,7 @@ Deploying Django
 
 Django's chock-full of shortcuts to make Web developer's lives easier, but all
 those tools are of no use if you can't easily deploy your sites. Since Django's
-inception, ease of deployment has been a major goal. There's a number of good
-ways to easily deploy Django:
+inception, ease of deployment has been a major goal.
```
Comment (MarkusH) [inline docs/howto/deployment/index.txt]: I'm not sure about dropping this sentence.

**Comment by MarkusH** [comment]
:+1: for dropping the "Django Book" reference.

File: docs/howto/deployment/index.txt (hunk 1 above)
Comment (yamila-moreno) [inline docs/howto/deployment/index.txt]: I understood it was required from the bug description; should I undo this change?

Comment (MarkusH) [inline docs/howto/deployment/index.txt]: Just discussed it with @freakboy3742 . We're going to drop that sentence. Thanks.

**Comment by MarkusH** [comment]
Merged in f954c517203d1fceaa1de26b17b242690cc138fa. Thank you!

**Comment by yamila-moreno** [comment]
UAU!! :dancers: :dancer: My first PR in Django ^_^ Thanks a lot!
